CefSharp.Core/Internals/ClientAdapter.cpp

```diff
@@ -195,8 +195,14 @@ namespace CefSharp
             {
                 return false;
             }
+            System::Boolean isKeyboardShortcut(*is_keyboard_shortcut);
```

Comment:
I didn't think it was necessary to do anything more than pass in `*is_keyboard_shortcut` as a `ref` param?

At least it appeared to be working when I added `no_javascript_access` to `OnBeforePopup`

https://github.com/cefsharp/CefSharp/pull/935/files#diff-2731f23ba4e8fcf96f8501a49c6d687eR53